From 345883e9f7de859619c7d7e7fbea60c369d31462 Mon Sep 17 00:00:00 2001
From: Matthew Quijano <mq022002@gmail.com>
Date: Fri, 15 Mar 2024 16:53:05 -0400
Subject: [PATCH 11/42] refactor: Add proficiency field

---
 firebase_convert.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/firebase_convert.py b/firebase_convert.py
index bfb532e..ebab48a 100644
--- a/firebase_convert.py
+++ b/firebase_convert.py
@@ -27,7 +27,7 @@ def to_camel_case(text):
 #     return title
 
 
-with open("data/topics2.json") as f:
+with open("data/topics3.json") as f:
     data = json.load(f)
 
 for topic_details in data["topics"]:
@@ -62,5 +62,6 @@ for topic_details in data["topics"]:
                     "experimentalGroupImageURLs", []
                 ),
                 "order": chapter_details["order"],
+                "proficiency": chapter_details["proficiency"],
             }
         )
-- 
2.38.1.windows.1

